src/uportQRDisplay.js

```diff
@@ -1,26 +0,0 @@
-import { Uport } from './uport'
-import { QRUtil } from './util/qrdisplay'
-
-
-class UportQRDisplay extends Uport {
-  constructor(dappName, opts = {}) {
-    super(dappName, opts)
-    this.openQR = QRUtil.openQr
-    this.closeQR = QRUtil.closeQr
-  }
-
-  request({uri, topic}) {
-    // TODO need self reference
-    console.log(this)
-    this.openQR(uri)
-    return new Promise((resolve, reject) => {
-      this.msgServer.waitForResult(topic, (err, res) => {
-        this.closeQR()
-        if (err) reject(err)
-        resolve(res)
-      })
-    })
-  }
-}
-
-export { UportQRDisplay }
```

Comment:
the main issue for me was the closeQr, it was making it difficult to loosely couple the display and data. I would be hesitant to have different build processes change the functionality of the code, which is hard not to do if they are more tightly coupled (closeQr), but yes which ever choice is fine right now